components/brave_webtorrent/extension/components/torrentViewerHeader.tsx

```diff
@@ -54,7 +54,7 @@ export default class TorrentViewerHeader extends React.PureComponent<
     const mainButtonText = torrent ? 'Stop Torrent' : 'Start Torrent'
     const copyButtonText = this.props.torrentId.startsWith('magnet:')
       ? 'Copy Magnet Link'
-      : 'Save File'
+      : 'Save .torrent File'
```

Comment:
probably we would like to add translations to torrent page

cc @rebron

Reply:
@NejcZdovc captured with https://github.com/brave/brave-browser/issues/858

Reply:
thx